onebusaway-android/src/main/java/org/onebusaway/android/report/ui/Open311ProblemFragment.java

```diff
@@ -492,16 +498,27 @@ private String getTransitIssueParameters(Service service) {
             // Append trip service params
             sb.append(getResources().getString(R.string.ri_append_stop_id, obaStop.getId()));
             if (mArrivalInfo != null) {
+                DateFormat dateFormat = new SimpleDateFormat("MM-dd-yyyy");
+
                 sb.append(getResources().getString(R.string.ri_append_trip_name, mArrivalInfo.getHeadsign()));
                 sb.append(getResources().getString(R.string.ri_append_route_id, mArrivalInfo.getRouteId()));
+                if (!TextUtils.isEmpty(mArrivalInfo.getShortName())) {
+                    sb.append(getResources().getString(R.string.ri_append_route_short_name,
+                            mArrivalInfo.getShortName()));
+                } else if (!TextUtils.isEmpty(mArrivalInfo.getRouteLongName())) {
+                    sb.append(getResources().getString(R.string.ri_append_route_long_name,
+                            mArrivalInfo.getRouteLongName()));
+                }
```

Comment:
Do you have a reference to `ObaRoute` here?  If so, let's use the `UIUtils.getRouteDisplayName(ObaRoute route)` method for this - this is used elsewhere in the app.  If not, what you have is fine.